public/chat_header_button.tsx

```diff
@@ -229,13 +233,33 @@ export const HeaderChatButton = (props: HeaderChatButtonProps) => {
     };
   }, [appId, flyoutVisible, props.assistantActions, registry]);
 
+  const loadLatestConversation = () => {
+    setFlyoutVisible(!flyoutVisible);
```

Comment:
I'm prefer to call `core.services.conversationLoad.status$.next('loading');` here. Then we can remove the `showWelcomePage` state.